psc-docs/Ctags.hs

```diff
@@ -1,5 +1,6 @@
 module Ctags (dumpCtags) where
 
+import Prelude
```

Comment:
Right. forgot to mention. This was added because intero gave a bunch of errors for not being able to find the functions from `Prelude` so I thought explicit is better than implicit and this might help others who try to work on this with intero in the future.